recipes/qt/6.x.x/conanfile.py

```diff
@@ -788,7 +788,8 @@ def _create_plugin(pluginname, libname, type, requires):
             _create_module("QuickTest", ["Test"])
 
         if self.options.qttools and self.options.gui and self.options.widgets:
-            _create_module("LinguistTools", ["Gui", "Widgets"])
+            self.cpp_info.components["qtLinguistTools"].names["cmake_find_package"] = "LinguistTools"
+            self.cpp_info.components["qtLinguistTools"].names["cmake_find_package_multi"] = "LinguistTools"
```

Comment:
should it link to threads like https://github.com/conan-io/conan-center-index/pull/6877#issuecomment-900994113? or is that handled else where?

Reply:
The comment is for the platform module which I think is platform specific. I would say we fix linguist first and worry about platform later as I'm not sure how complex that will be to tackle.